docs/usage/rust.md

```diff
@@ -19,11 +19,18 @@ Renovate supports upgrading dependencies in `Cargo.toml` files and their accompa
 
 Renovate updates Rust crates by default.
 
-## Private crate registries and private Git dependencies
+## Private Git crate registries and private Git dependencies
 
 If any dependencies are hosted in private Git repositories, [Git Authentication for cargo](https://doc.rust-lang.org/cargo/appendix/git-authentication.html) must be set up.
 
-If any dependencies are hosted on private crate registries (i.e., not on `crates.io`), then credentials should be set up in such a way that the Git command-line is able to clone the registry index.
+If any dependencies are hosted on private Git crate registries (i.e., not on `crates.io`), then credentials should be set up in such a way that the Git command-line is able to clone the registry index.
 Third-party crate registries usually provide instructions to achieve this.
 
 Both of these are currently only possible when running Renovate self-hosted.
+
+## Private sparse crate registries
+
+Renovate can update dependencies hosted on a private sparse crate registry (<https://doc.rust-lang.org/beta/cargo/reference/registry-index.html#sparse-protocol>).
+Since sparse registries are HTTP based, authentication for Renovate can be configured via additional hostRules.
+
+Renovate can only update Cargo lockfiles for projects using dependencies from sparse registries if the local Rust toolchain has the sparse-registry feature enabled (i.e beta/nightly/or 1.68 or later - 1.68 is due for release on March 9th 2023).
```

Comment:
```suggestion
Before Renovate can update Cargo lockfiles for projects using dependencies from sparse registries you must:

- Use Rust `1.68` or later
- Enable the `sparse-registry` feature in your local Rust toolchain
```

How about something like this?

The current sentence is too long, and has too much information in one go. So we should split things up into smaller parts. 😉

Reply:
> Perhaps omit all mention of the sparse-registry feature, which is actually a compilation flag so unlikely something most users can control?

It sounds we have two groups of people here:

1. General public who uses Renovate to update Rust dependencies
1. Maintainers of Rust repositories who want Renovate to work properly for their published packages

Sounds like the compilation flag `sparse-registry` is aimed at number 2 then?

I'll let the Renovate maintainers decide what to keep in the docs and what to drop. 😉 

> The only oddity with that is that Rust 1.68 isn't out yet, but anyone using sparse registries before then is likely using a beta/nightly toolchain, or a self-built with sparse-registry built in (I fall into this camp :)), and can figure out how to get it working.

I'm okay with mentioning the Rust version `1.68` ahead of time. Are you sure that the `sparse-registry` feature will be included in version `1.68`?

Reply:
> Are you sure that the sparse-registry feature will be included in version 1.68?

Yes, at least that's still the current plan: https://blog.rust-lang.org/inside-rust/2023/01/30/cargo-sparse-protocol.html